content/en/docs/contributing/docs.md

```diff
@@ -16,10 +16,200 @@ You can add entire sections if you think a topic is missing or update existing o
+
+**You can also run the linter locally.
+Note that there is a set of linters run via `super-linter`.**
+
+For the Unikraft documentation repository, used and useful linters are:
+
+- [`markdown-cli`](https://github.com/igorshubovych/markdownlint-cli#readme)
+- [`textlint`](https://textlint.github.io/)
+- [`jscpd`](https://github.com/kucherenko/jscpd)
+
+To run the `super-linter` locally, follow the instructions from [here](https://github.com/github/super-linter/blob/main/docs/run-linter-locally.md).
+Note that this will require quite a lot of storage, since it will pull the `super-linter` Docker image.
+
+You will need to install Docker, following the [official documentation](https://docs.docker.com/install/).
+After that, pull the latest container by running:
+
+```console
+$ docker pull github/super-linter:latest
+```
+
+To run the `super-linter` for one file, use:
+
+```console
+$ docker run -e RUN_LOCAL=true -e USE_FIND_ALGORITHM=true -v /path/to/local/codebase/file:/tmp/lint/file github/super-linter
+--------------------------------------------------------------------------------
+2022-12-29 07:50:55 [INFO]   ---------------------------------------------
+2022-12-29 07:50:55 [INFO]   --- GitHub Actions Multi Language Linter ----
+2022-12-29 07:50:55 [INFO]    - Image Creation Date:[2022-12-26T16:46:33Z]
+2022-12-29 07:50:55 [INFO]    - Image Revision:[154522b380449499473f75d46f3796e23ebc1d2e]
+2022-12-29 07:50:55 [INFO]    - Image Version:[154522b380449499473f75d46f3796e23ebc1d2e]
+2022-12-29 07:50:55 [INFO]   ---------------------------------------------
+2022-12-29 07:50:55 [INFO]   ---------------------------------------------
+2022-12-29 07:50:55 [INFO]   The Super-Linter source code can be found at:
+2022-12-29 07:50:55 [INFO]    - https://github.com/github/super-linter
+2022-12-29 07:50:55 [INFO]   ---------------------------------------------
+2022-12-29 07:50:55 [INFO]   --------------------------------------------
+[...]
+```
+
+You can also run each linter separately, since it will require less storage space.
+To run the linters, you will need to install the [`markdown-cli`](https://github.com/igorshubovych/markdownlint-cli), [`textlint`](https://textlint.github.io/docs/getting-started.html) and [`jscpd`](https://github.com/kucherenko/jscpd#installation) tools:
+
+```console
+$ npm install -g markdownlint-cli
+$ npm install -g textlint
+$ npm install -g jscpd
+```
+
+The `textlinter` will require installation of rules.
+You can find a list of rules [here](https://github.com/textlint/textlint/wiki/Collection-of-textlint-rule).
+
+```console
+$ npm install -g textlint-rule-no-todo
+```
+
+To run the linters, from inside the directory you want to check, use:
+
+```console
+$ markdownlint --config <path-to-your-docs-repo-clone>/.github/workflows/config/config.json --rules <path-to-your-docs-repo-clone>/.github/workflows/rules/rules.js *.md
+$ npx textlint --rule <rule-name> <file>
+$ npx jscpd <path-to-source-code>
+```
```

Comment:
Add sample runs, concrete commands and their output, to let readers know what they should fill the `<path-to-source-code>` argument with. And to have a feel of what the output should look like.